doc/architecture/20170824-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,36 @@
+## Problem 
+
+1. Discrepancies in the expected numbers of content items that need to be audited when we compare them with the existing spreadsheets. 
+2. Themes do not represent the input of an Audit.
+  
+## Decision
+
+1. Introduce the notion of an `Audit Plan` (defined below) that models the input of an audit.
+2. Do not use Themes in the Auditing tool, and rely on primary organisations.
+
+## Audit plans
+
+An audit plan is a set of content items that need auditing by a team of Content Auditors. 
+
+It is conformed of:
+
+- A set of subscription rules 
+- A set of formats to exclude
+- A set of content items to include
+- A set of content items to exclude
+
+As of now, as we are relying on organisations, we will limit the subscription rules to a single rule that represents the content that belongs an organisation.
+ 
+## Terms
+
+The following list represent represent concepts that have been used in the process to define an inventory. 
+We are writing them down here as they have been a source of misunderstanding for the team.
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting; It is subject related, and it attempts to explain an approximation of the 'aboutness' of content items that would reside in a specific theme 
```

Comment:
~~GOV.Uk~~ GOV.UK

Reply:
Sentence missing a full stop.

Reply:
Addressed in d509fe718b81d8e902dd7a0f71f8f00df099a49a